pkg/kotsutil/kots.go

```diff
@@ -416,6 +416,42 @@ func LoadKotsKindsFromPath(fromDir string) (*KotsKinds, error) {
 	return &kotsKinds, nil
 }
 
+func LoadHelmChartsFromPath(fromDir string) ([]*kotsv1beta1.HelmChart, error) {
+	charts := []*kotsv1beta1.HelmChart{}
+	decode := scheme.Codecs.UniversalDeserializer().Decode
+	err := filepath.Walk(fromDir,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if info.IsDir() {
+				return nil
+			}
+
+			contents, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			decoded, gvk, err := decode(contents, nil, nil)
+			if err != nil {
+				return nil
```

Comment:
wrap this with the path since its in a loop

Reply:
not an error